_contrib/source/index.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Have fun ❤︎</title>
+    <link rel="stylesheet" href="assets/stylesheets/application.css">
+    <script type="text/javascript" src="assets/javascripts/application.js"></script>
+  </head>
+  <body class="container">
+    <br>
```

Comment:
Could you use a margin top in lieu of `br`s?

Reply:
Typically I would but this is not meant to be functioning html and only exists as an example page that one can develop against. As such it doesn't have any css for itself, just that of the library.